src/commands/__tests__/extractThought.ts

```diff
@@ -68,21 +80,25 @@ describe.skip('Extract thought', () => {
     expect(createdThought).toBeTruthy()
 
     // created thought gets appended to the end
-    const thoughtChildrenWrapper = thought!.closest('li')?.lastElementChild as HTMLElement
-    const thoughtChildren = await findAllByPlaceholderText(thoughtChildrenWrapper, 'Add a thought')
+    const thoughtChildrenWrapper = thought!.closest('div[aria-label=tree-node]')?.lastElementChild as HTMLElement
+    const thoughtChildren = await findAllByLabelText(thoughtChildrenWrapper, 'thought')
+    await timer.runAllAsync()
 
-    expect(thoughtChildren.map((child: HTMLElement) => child.textContent)).toMatchObject(['sub-thought', 'thought'])
+    expect(thoughtChildren.map((child: HTMLElement) => child.textContent)).toMatchObject(['this is a thought'])
```

Comment:
If a thought with the text `this is a thought` still exists, then `extractThought` didn't work. It would have extracted "thought".